spec/fortnox/api/repositories/examples/save_examples.rb

```diff
@@ -0,0 +1,41 @@
+shared_examples_for '.save' do |attribute, fortnox_attribute|
```

Comment:
I don't think that `attribute` and `fortnox_attribute` are good names. I'm guessing that `attribute` should be `attribute_name` or some such? And for `fortnox_attribute` I'd go with `attribute_json_name` ... And that leads me to want them both at the same level of specificity so then I'd change `attribute` to `attribute_hash_name` or `attribute_hash_key_name` instead.

The main point of these shared examples is that since they are out of context when you read them (the incoming actual data is in another file) you need to be even more careful in how you name things and generally opt for more specific names when you can.